Kitodo/src/main/java/de/sub/goobi/config/Parameters.java

```diff
@@ -0,0 +1,695 @@
+
+    /**
+     * If no Accept-Language Http Request header is present, use the following
+     * language.
+     */
+    public static final String LANGUAGE_DEFAULT = "language.default";
+
+    // -----------------------------------
+    // Data protection
+    // -----------------------------------
+
+    /**
+     * The General Data Protection Regulation or local law might require to set
+     * this value to true. anonymized statistics, displaying user on steps, etc.
+     * Boolean, defaults to {@code false}.
+     */
+    public static final String ANONYMIZE = "anonymize";
+
+    /**
+     * Enable / disable search for steps done by user. Boolean, defaults to
+     * {@code false}.
+     */
+    public static final String WITH_USER_STEP_DONE_SEARCH = "withUserStepDoneSearch";
+
+    // -----------------------------------
+    // Error page
+    // -----------------------------------
+
+    /**
+     * Page the user will be directed to continue.
+     */
+    public static final String ERR_LINK_TO_PAGE = "err_linkToPage";
+
+    // =============================================================================
+    // METADATA PROCESSING
+    // =============================================================================
+    // -----------------------------------
+    // Catalogue search
+    // -----------------------------------
+
+    /**
+     * Number of hits to show per page on the hitlist when multiple hits were
+     * found on a catalogue search. Integer, defaults to 12.
+     */
+    public static final String HITLIST_PAGE_SIZE = "catalogue.hitlist.pageSize";
+
+    /**
+     * Indicates the maximum duration an interaction with a library catalogue
+     * may take. Milliseconds, defaults to 30 minutes.
+     */
+    public static final String CATALOGUE_TIMEOUT = "catalogue.timeout";
+
+    // -----------------------------------
+    // Metadata editor behavior
+    // -----------------------------------
+    /**
+     * Long, value in milliseconds. Defaults to 180000 (30 minutes).
+     */
+    public static final String METS_EDITOR_LOCKING_TIME = "MetsEditorLockingTime";
+
+    /**
+     * Use special image folder for METS editor if exists (define suffix here).
+     */
+    public static final String METS_EDITOR_DEFAULT_SUFFIX = "MetsEditorDefaultSuffix";
+
+    /**
+     * Enables or disables automatic pagination changes in the meta-data editor.
+     * If false, pagination must be updated manually by clicking the link “Read
+     * in pagination from images”. Boolean, defaults to {@code true}.
+     */
+    public static final String WITH_AUTOMATIC_PAGINATION = "MetsEditorWithAutomaticPagination";
+
+    /**
+     * Use special pagination type for automatic default pagination.
+     */
+    public static final String METS_EDITOR_DEFAULT_PAGINATION = "MetsEditorDefaultPagination";
+    public static final String METS_EDITOR_DEFAULT_SUFFIX_VALUE_ARABIC = "arabic";
```

Comment:
Here you are mixing config keys and values. Shouldn't they be separated?

Reply:
Same as above.